node/nodeInternalBlocks_test.go

```diff
@@ -36,8 +38,10 @@ func TestCreateInternalBlockProcessor_NilStoreShouldErr(t *testing.T) {
 		node.WithDataComponents(dataComponentsMock),
 	)
 
+	expectedErr := errors.New(
+		"error creating transaction status computer " + txstatus.ErrNiStorageService.Error())
 	err := n.CreateInternalBlockProcessor()
-	assert.Error(t, err)
+	assert.Equal(t, err, expectedErr)
```

Comment:
you can check with something like:
assert.True(t, errors.Is(err, data.ErrNiStorageService))
then you don't need the expectedErr creation

Reply:
done